src/views/TradingCompetition/components/PrizesInfo/PrizesCard.tsx

```diff
@@ -28,8 +28,10 @@ const PrizesCard = () => {
       <Box p="24px">
         <Text color="textSubtle" fontSize="14px">
           {t(
-            'Prizes to be distributed in CAKE and shared by all members of a tier. CAKE price in USD to be determined on the day of distribution. Details below.',
-          )}
+            'Prizes to be distributed in CAKE, LAZIO, PORTO and SANTOS in a distribution of 3:1:1:1 and shared by all members of each respective tier.',
+          )}{' '}
+          {t('Token prizes in Fan Tokens will be distributed as per their BUSD pair price on the day of registration.')}{' '}
```

Comment:
@chef-jojo isn't it supposed to be the day of distribution?
Also registration is 3 days 👀 
Let's discuss in slack...